ta/pkcs11/src/pkcs11_attributes.h

```diff
@@ -162,6 +163,12 @@ check_mechanism_against_processing(struct pkcs11_session *session,
 				   enum processing_func function,
 				   enum processing_step step);
 
+static inline bool attribute_is_hidden(struct pkcs11_attribute_head *req_attr)
+{
+	return (req_attr->id & PKCS11_CKA_OPTEE_FLAGS_HIDDEN) ==
```

Comment:
`(req_attr->id & PKCS11_CKA_OPTEE_FLAGS_MASK) == PKCS11_CKA_OPTEE_FLAGS_HIDDEN`

Reply:
If we would introduce new flags then this would start to fail.

Reply:
Sorry, indeed `PKCS11_CKA_OPTEE_FLAGS_HIDDEN` already consider `PKCS11_CKA_VENDOR_DEFINED` flags. Discard my comment, this looks good to me.